pkg/framework/test/apiserver_config.go

```diff
@@ -4,7 +4,7 @@ import "github.com/asaskevich/govalidator"
 
 // APIServerConfig is a struct holding data to configure the API Server process
 type APIServerConfig struct {
-	EtcdURL      string `valid:"required,url"`
+	EtcdURL      string `valid:"url"`
```

Comment:
Can this be the Etcd object instead?

Reply:
Also, is this used? Since you connect to `etcd.GetURL` rather than this value now?